### PR TITLE
fix display errors

### PR DESCRIPTION
**Pull request by zhouyun-zoe**

### CODE REVIEW

**Comment by kaikai1024** [comment]
I'm not sure if it works.

**Review by kaikai1024 — DISMISSED** [review]

**Review by boundless-forest — DISMISSED** [review]

File: docs/cita/getting-started.md [hunk 1]
```diff
@@ -231,7 +230,7 @@ $ bin/cita help
 > * 请勿在一台服务器上运行多个容器。因为虽然 CITA 在 Docker 中运行，但是容器并没有做网络隔离。
 > * 请不要同时在 host 系统里面运行 CITA 以及相关的 RabbitMQ 等软件，以免造成端口冲突。
 
-## 验证
```
Comment (kaikai1024) [inline docs/cita/getting-started.md]: Should search the link of this and replace it.

File: docs/cita/getting-started.md [hunk 2]
```diff
@@ -203,15 +202,15 @@ $ bin/cita create --super_admin "0x141d051b1b1922bf686f5df8aad45cefbcb0b696" --n
 
    更多信息请查看[验证](#验证)。
```
Comment (leeyr338) [inline docs/cita/getting-started.md:203]: 这个链接在下面删除了？

Comment (zhouyun-zoe) [reply docs/cita/getting-started.md:203]: fixed

File: docs/cita/getting-started.md (hunk 1 above)
Comment (zhouyun-zoe) [reply docs/cita/getting-started.md]: fixed

**Comment by kaikai1024** [comment]
ignore `website/i18n/en.json`

**Review by leeyr338 — APPROVED** [review]

**Comment by zhouyun-zoe** [comment]
> ignore `website/i18n/en.json`

ok！

**Pull request merged**